### PR TITLE
TXAD-168: Implement Delete Document and Delete Item API.

### PR DESCRIPTION
**Pull request by ruchawajpe**

### CODE REVIEW

File: lib/screens/workorder/widgets/workorder_item_tab_delete_button.dart [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    return CustomIconButton(
+        icon: Icons.delete,
+        onPressed: () {
+          showDialog(
+              context: context,
+              builder: (context) {
+                return AndroidPopUp(
+                    titleValue: DatabaseUtil.getText('DeleteRecord'),
+                    contentValue: '',
+                    onPrimarybtn: () {
```
Comment (Aditi-005) [inline lib/screens/workorder/widgets/workorder_item_tab_delete_button.dart:28]: Reformat this to onPrimaryButton

File: lib/screens/workorder/widgets/workorder_tab_three_items_tab.dart [hunk 2]
```diff
@@ -9,12 +9,13 @@ import '../../../data/models/workorder/fetch_workorder_details_model.dart';
...
 import '../../../widgets/custom_icon_button.dart';
+import 'workorder_item_tab_delete_button.dart';
 
 class WorkOrderTabThreeItemsTab extends StatelessWidget {
   final WorkOrderDetailsData data;
 
-  const WorkOrderTabThreeItemsTab({Key? key, required this.data})
-      : super(key: key);
+  WorkOrderTabThreeItemsTab({Key? key, required this.data}) : super(key: key);
+  final Map deleteItemTabItemMap = {};
```
Comment (Aditi-005) [inline lib/screens/workorder/widgets/workorder_tab_three_items_tab.dart:18]: where are we using this map?

**Review by Aditi-005 — CHANGES_REQUESTED** [review]

File: lib/screens/workorder/widgets/workorder_item_tab_delete_button.dart (hunk 1 above)
Comment (ruchawajpe) [reply lib/screens/workorder/widgets/workorder_item_tab_delete_button.dart:28]: Done

File: lib/screens/workorder/widgets/workorder_tab_three_items_tab.dart (hunk 2 above)
Comment (ruchawajpe) [reply lib/screens/workorder/widgets/workorder_tab_three_items_tab.dart:18]: Removed.

**Review by Aditi-005 — APPROVED** [review]

**Pull request merged**